src/script/interpreter.cpp

```diff
@@ -1431,8 +1431,8 @@ void PrecomputedTransactionData::Init(const T& txTo, std::vector<CTxOut>&& spent
     }
 
     // Determine which precomputation-impacting features this transaction uses.
-    bool uses_bip143_segwit = false;
-    bool uses_bip341_taproot = false;
+    bool uses_bip143_segwit = force;
+    bool uses_bip341_taproot = force;
     for (size_t inpos = 0; inpos < txTo.vin.size(); ++inpos) {
```

Comment:
in 474536660d:

May as well skip this `for` loop altogether if `force` is true, if I see that correctly.

Reply:
Done.